azure-pipelines.yml

```diff
@@ -57,6 +57,15 @@ jobs:
     configuration: Debug
     testArguments: -testCoreClr
 
+- template: eng/pipelines/test-windows-job-single-machine.yml
```

Comment:
Do we want the extra jobs or should we just have the one run of coreclr debug tests on single machine?

Reply:
I think it's nice to have the single machine and the Helix runs going to allow us to compare them for now.